lib/active_merchant/billing/gateways/decidir_plus.rb

```diff
@@ -144,6 +144,35 @@ def add_sub_payments(post, options)
         end
       end
 
+      def add_payment_method_id(options)
```

Comment:
Is it worth adding a unit test to cover the behavior of this method?

I'm inclined to say no, but I just wanted to bring it up in case you like the idea.

Reply:
I thought about adding more test, but the urgency to get this in is rising, so I'm just working towards expediency.